### PR TITLE
HDDS-8330. [Doc] Fix misplaced doc for GDPR.zh.md

### PR DESCRIPTION
**Pull request by whbing**
## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-8330

### CODE REVIEW

File: hadoop-hdds/docs/content/security/GDPR.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 title: "GDPR in Ozone"
 date: "2019-September-17"
 weight: 3
-summary: GDPR in Ozone
```
Comment (kaijchen) [inline hadoop-hdds/docs/content/security/GDPR.md:5]: Why is this line removed?

File: hadoop-hdds/docs/content/security/SecuringS3.zh.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: "安全化 S3"
 date: "2019-April-03"
 summary: Ozone 支持 S3 协议，并使用 AWS Signature Version 4 protocol which allows a seamless S3
  experience.
-weight: 4
+weight: 5
```
Comment (kaijchen) [inline hadoop-hdds/docs/content/security/SecuringS3.zh.md:6]: Why do we need to change weights?

**Review by kaijchen — COMMENTED** [review]
Thanks @whbing for the patch. The menu change looks good. But is other changes necessary?

File: hadoop-hdds/docs/content/security/GDPR.md (hunk 1 above)
Comment (whbing) [reply hadoop-hdds/docs/content/security/GDPR.md:5]: > Why is this line removed?

@kaijchen  `summary ` repeated.
The line deleted was the same as the `title`, so I kept the latter.
```js
summary: GDPR in Ozone
icon: user
menu:
   main:
      parent: Security
summary: Support to implement the "Right to be Forgotten" requirement of GDPR
```

File: hadoop-hdds/docs/content/security/SecuringS3.zh.md (hunk 2 above)
Comment (whbing) [reply hadoop-hdds/docs/content/security/SecuringS3.zh.md:6]: > Why do we need to change weights?

`*.zh.md` and `*.md` are not in the same order on page, just  consistent with `*.md`

**Review by kaijchen — APPROVED** [review]
LGTM, thanks @whbing.
